src/views/communities/CommunityCard.tsx

```diff
@@ -30,184 +25,248 @@ function CommunityCard(props: { community: CommunityAttributes }) {
+                    </Text>
+                    <Text style={styles.cardCommunityDescription}>
+                        {community.description?.length > 100
+                            ? community.description.substr(0, 100) + '...'
+                            : community.description}
+                    </Text>
                     <View
                         style={{
-                            flex: 3,
                             flexDirection: 'row',
-                            marginTop: 19.27,
-                            marginBottom: 17.85,
+                            alignItems: 'center',
+                            bottom: 0,
                         }}
                     >
-                        <View style={{ flex: 1 }}>
-                            <Text style={styles.cellHeader}>
+                        <View
+                            style={{
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                                flexDirection: 'row',
+                                marginRight: 12,
+                            }}
+                        >
+                            <BeneficiariesSvg style={{ marginRight: 4 }} />
+                            <Text style={styles.bottomCardText}>
                                 {community.state.beneficiaries}
                             </Text>
-                            <Text style={styles.cellDescription}>
-                                {i18n.t('generic.beneficiaries')}
-                            </Text>
                         </View>
-                        <View style={{ flex: 1 }}>
-                            <Text style={styles.cellHeader}>
-                                $
-                                {humanifyCurrencyAmount(
-                                    community.contract.claimAmount
-                                )}
-                            </Text>
-                            <Text style={styles.cellDescription}>
-                                {claimFrequencyToText(
-                                    community.contract.baseInterval
+                        <View
+                            style={{
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                                flexDirection: 'row',
+                                marginRight: 12,
+                            }}
+                        >
+                            <Text style={styles.bottomCardText}>
+                                ~
+                                {amountToCurrency(
+                                    community.contract.claimAmount,
+                                    community.currency,
+                                    rates
                                 )}
                             </Text>
+                            <Text style={styles.interPunct}>{'\u2B24'}</Text>
```

Comment:
Assuming this is the dot, first it's on the wrong place, second, why not just using the already used in other views Dot component https://github.com/impactMarket/mobile-app/blob/master/src/components/Dot.tsx?

Reply:
That's NOT a dot (bottom alignment w/ texts) but an interPunct (middle alignment with texts). It's more like a bullet character, used only in this place where it's now. That's the reason I didn't created a separated component for that.

Reply:
> used only in this place where it's now

?

Reply:
Reviewed. Fixed.